lib/routes/boc/whpj.js

```diff
@@ -43,8 +43,8 @@ module.exports = async (ctx) => {
         .map(function () {
             const zh_name = $(this).find('td:nth-child(1)').text();
             const en_name = en_names[zh_name] || '';
-            const name = `${zh_name} ${en_name} `;
-            const date = `${$(this).find('td:nth-child(7)').text()} ${$(this).find('td:nth-child(8)').text()}`;
+            const name = `${zh_name} ${en_name}`;
+            const date = String($(this).find('td:nth-child(7)').text());
```

Comment:
```suggestion
            const date = $(this).find('td:nth-child(7)').text();
```
`.text()` [returns a string](https://cheerio.js.org/docs/api/classes/Cheerio#text). No need to cast it again.

Reply:
Thanks! The auto formatter changed `${...}` to `String(...)` and I didn't notice this